src/auth/oauth.ts

```diff
@@ -554,4 +582,60 @@ export class OAuth extends BaseAuthAPI {
 
     return VoidApiResponse.fromResponse(response);
   }
+
+  /**
+   * Exchanges a subject token (e.g. a refresh token) for a federated connection access token.
+   *
+   * The request body includes:
+   * - client_id (and client_secret via addClientAuthentication)
+   * - grant_type set to FCAT_GRANT_TYPE
+   * - subject_token and fixed subject_token_type value for refresh tokens
```

Comment:
Let's also add the value for `subject_token_type` if we are adding jsdocs like this.